lib/api/helpers.js

```diff
@@ -200,6 +201,33 @@ export function getSlug(slugString) {
   return slug;
 }
 
+/**
+ * @name formatPhoneNumber
+ * @method
+ * @memberof Helpers
+ * @summary prepends country code to phone no. if required
+ * @param  {String} phone the original phone no.
+ * @param  {String} countryCode the country's code to which the phone no. belongs to
+ * @return {String} the phone no. with country extension.
+ */
+export function formatPhoneNumber(phone, countryCode) {
+  try {
+    // phone no. already has the country code attached
+    if (isValidNumber(phone)) {
+      return phone;
+    }
+    const phoneNumber = parse(phone, countryCode).phone;
+    const countryExtension = getPhoneCode(countryCode);
+    if (phoneNumber && countryExtension) {
+      // try attaching the country code to phone number
+      return `${countryExtension}${phoneNumber}`;
+    }
+    throw new Meteor.Error("invalid-parameter", "Incorrect format for phone number.");
+  } catch (error) {
```

Comment:
Should we maybe put the actual error code in the error output since "Incorrect format for phone number" may not actually be the error here?

Reply:
Hmm, yes makes sense. Doing that.